uber/utils.py

```diff
@@ -108,15 +108,25 @@ def check_csrf(csrf_token):
     """
     if csrf_token is None:
         csrf_token = cherrypy.request.headers.get('CSRF-Token')
+
     if not csrf_token:
         raise CSRFException("CSRF token missing")
+
     if csrf_token != cherrypy.session['csrf_token']:
-        log.error("csrf tokens don't match: {!r} != {!r}", csrf_token, cherrypy.session['csrf_token'])
-        raise CSRFException('CSRF check failed')
```

Comment:
mostly cosmetic, changed the logging so it puts the message into the exception so we see it in our handlers